tests/steamship_tests/app/integration/test_package_instance.py

```diff
@@ -135,6 +135,19 @@ def get_raw(path: str):
         resp_obj = instance.invoke("json_with_status", verb=Verb.POST)
         assert resp_obj == {"status": "a string"}
 
+        # Test that the __steamship_dir__ method works
+        #
+        # Note: The output of the InvocableResponse includes a dynamically generated value:
+        #
+        #       returns': "<class 'l_dd6730cb.InvocableResponse[bytes]'>"
```

Comment:
Could regex replace the dynamically generated value in both the test asset and the test for comparison.

Reply:
Ooh that's a good idea. I'll do that now. Just didn't have the gas left in my tank before to finish the job.